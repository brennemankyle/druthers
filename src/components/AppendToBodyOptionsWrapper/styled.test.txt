// import React from "react";
// import renderer from "react-test-renderer";
// import { mockStyles, mockRect } from "../../mocks";
// import { StyledAppendToBodyOptionsWrapper } from "./styled";

// it("styles", () => {
//   const wrapper = renderer.create(
//     <StyledAppendToBodyOptionsWrapper {...mockStyles} parentRect={mockRect} />
//   );

//   expect(wrapper).toMatchSnapshot();
// });

// it("placeOptionsAbove = true", () => {
//   const wrapper = renderer.create(
//     <StyledAppendToBodyOptionsWrapper
//       {...mockStyles}
//       parentRect={mockRect}
//       placeOptionsAbove={true}
//     />
//   );

//   expect(wrapper).toMatchSnapshot();
// });

// it("styles_rightToLeft = true", () => {
//   const wrapper = renderer.create(
//     <StyledAppendToBodyOptionsWrapper
//       {...mockStyles}
//       parentRect={mockRect}
//       styles_rightToLeft={true}
//     />
//   );

//   expect(wrapper).toMatchSnapshot();
// });

// it("styles_hasOptions = false", () => {
//   const wrapper = renderer.create(
//     <StyledAppendToBodyOptionsWrapper
//       {...mockStyles}
//       parentRect={mockRect}
//       styles_hasOptions={false}
//       styles_multiple={false}
//     />
//   );

//   expect(wrapper).toMatchSnapshot();
// });
