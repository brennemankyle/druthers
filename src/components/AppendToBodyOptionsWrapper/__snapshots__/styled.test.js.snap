// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`placeOptionsAbove = true 1`] = `
.emotion-0 {
  position: absolute;
  background-color: white;
  width: 600px;
  bottom: 743px;
  left: 15px;
}

<div
  className="emotion-0"
/>
`;

exports[`styles 1`] = `
.emotion-0 {
  position: absolute;
  background-color: white;
  width: 600px;
  top: 125px;
  left: 15px;
}

<div
  className="emotion-0"
/>
`;

exports[`styles_hasOptions = false 1`] = `
.emotion-0 {
  position: absolute;
  background-color: white;
  width: 600px;
  top: 125px;
  left: 15px;
  display: none;
}

<div
  className="emotion-0"
/>
`;

exports[`styles_rightToLeft = true 1`] = `
.emotion-0 {
  position: absolute;
  background-color: white;
  width: 600px;
  top: 125px;
  left: 15px;
  -webkit-transform: scaleX(-1);
  -ms-transform: scaleX(-1);
  transform: scaleX(-1);
}

<div
  className="emotion-0"
/>
`;
